pkg/clustermanager/cluster_manager.go

```diff
@@ -1137,6 +1137,11 @@ func (c *ClusterManager) CreateEKSANamespace(ctx context.Context, cluster *types
 	return c.clusterClient.CreateNamespaceIfNotPresent(ctx, cluster.KubeconfigFile, constants.EksaSystemNamespace)
 }
 
+// CreatePackagesNamespace creates the eksa-packages namespace on the cluster if it doesn't already exist.
+func (c *ClusterManager) CreatePackagesNamespace(ctx context.Context, cluster *types.Cluster) error {
```

Comment:
curious, why is this needed now?
doesn't the "install packages" functionality (through helm) create this ns?

Reply:
We need the namespace before installing packages or else we can't add role/rolebindings in that ns